src/modules/task-manipulation.mod.js

```diff
@@ -0,0 +1,149 @@
+import { Tasks } from './task.mod.js';
+import { Alert } from './alerts.mod.js';
+
+export class TaskManipulation {
+  // Adds a new task to the list
+  static addTask = () => {
+    const description = document.getElementById('new-task').value;
+    if (description !== '') {
+      const add = Tasks.push({
+        index: Tasks.length,
+        description: description,
+        checked: false
+      });
+      // Save task to local storage
+      this.saveTask();
+
+      if (add) {
+        new Promise((resolve) => {
+          resolve(Alert.showSuccess('Task added successfully'));
+        });
+        return true;
+      }
+    }
+    else {
+      new Promise((resolve) => {
+        resolve(Alert.showError());
+      });
+      return false;
+    }
+  }
+
+  // Removes the task from the list
+  static removeTask = (e) => {
+    // Get the index of the task to be removed
+    const index = e.target.parentNode.parentNode.parentNode.getAttribute('data-taskindex');
+    // Remove the task from the list
+    const remove = Tasks.splice(index, 1);
```

Comment:
- [ ] Please kindly use array.filter method instead of splice as in the project requirement

Reply:
Thanks for your feedback. I have implemented this now. 👍🏽